language-configuration.json

```diff
@@ -15,14 +15,18 @@
         ["[", "]"],
         ["(", ")"],
         ["\"", "\""],
-        ["'", "'"]
+        ["'", "'"],
+        ["*", "*"],
```

Comment:
I don't really understand why auto close `*` and `_` (testing here it seemed strange -- can you elaborate a bit on the scenario for this?)

Reply:
This was just an idea because of the Documentation syntax. *bold* and _italic_ .

Maybe we can delete this from autoclose. You are right. This would be wierd everytime you press underscore somewhere and get two of them.

But surrounding pairs could be handy.

Reply:
This was just an idea because of the Documentation syntax. *bold* and _italic_ .

Maybe we can delete this from autoclose. You are right. This would be wierd everytime you press underscore somewhere and get two of them.

But surrounding pairs could be handy.

Reply:
Fixed